app/screens/Search/index.js

```diff
@@ -304,10 +327,12 @@ class Search extends Component {
       tabsData: [], searchedResult: [],
       sourceId: item.sourceId, languageCode: item.languageCode,
       languageName: item.languageName, versionCode: item.versionCode,
-      downloaded: item.downloaded, books: item.books
+      downloaded: item.downloaded, books: item.books,metadata:item.metadata
+
     })
   }
   render() {
+    console.log(" STATE Metadata ",this.state.metadata && this.state.metadata[0].revision)
```

Comment:
Remove console.log

Reply:
removed console from file and pushed code to same branch.